app/src/main/java/com/project/squirrelobserver/squirrelObserver/RecordActorTabActivity.java

```diff
@@ -41,6 +41,8 @@
     private ToggleButton previousButton = null;
     private final ArrayList<ToggleButton> list = new ArrayList<ToggleButton>();
     private final ArrayList<ToggleButton> listRecent = new ArrayList<ToggleButton>();
+    private final ArrayList<ToggleButton> disabledList = new ArrayList<>();
+    private final ArrayList<ToggleButton> disabledListRecent = new ArrayList<>();
```

Comment:
These don't really need to be final. list and listRecent are final because they are used to create an ArrayAdapter. Not a huge deal though